src/Uno.UI/UI/Xaml/UIElement.macOS.cs

```diff
@@ -292,23 +405,23 @@ public override void MouseUp(NSEvent evt)
 
 				if (!wasPointerOver && IsPointerOver)
 				{
-					pointerEventIsHandledInManaged = RaiseEvent(PointerEnteredEvent, args);
+					pointerEventIsHandledInManaged = RaisePointerEvent(PointerEnteredEvent, args);
```

Comment:
In the hypothesis that you use the [`OnNativePointerUp`](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/UIElement.Pointers.cs#L503), you can safely remove all the `<was|is>PointerOver` checks.

Reply:
Done